pt/lessons/basics/functions.md

```diff
@@ -89,6 +89,28 @@ iex> Length.of [1, 2, 3]
 3
 ```
 
+### Nomaear Funçoes e a Aridade
+
+Mencionamos anteriormente que as funçoes são nomeadas pela combinação da nome e aridade(quantidade dos argumentos) das funçoes. Isto significa que você pode fazer o siguiente.
+
+```elixir
+defmodule Greeter2 do
+  def hello(), do: "Hello, anonymous person!"   # hello/0
+  def hello(name), do: "Hello, " <> name        # hello/1
+  def hello(name1, name2), do: "Hello, #{name1} and #{name2}"
+                                                # hello/2
+end
+
+iex> Greeter2.hello()
+"Hello, anonymous person!"
+iex> Greeter2.hello("Fred")
+"Hello, Fred"
+iex> Greeter2.hello("Fred", "Jane")
+"Hello, Fred and Jane"
+```
+
+Nós temos listado os nomes das funçoes nos comentários acima. A causa que a primeira implementação não recebe argumentos, é conhecido como `hello/0`; a segunda função recebe um argumento, portanto será conhecido como `hello/1` e assim por diante. E ao contrário de sobrecargar funçoes como outros idiomas, estas são pensados como funçoes diferentes entre si. (Pattern matching, o a combinaçao de padrões, descrita só um momento atrás, aplica-se quando várias definiçoes são fornecidas com o mesmo quantidade de argumentos.)
```

Comment:
maybe `E ao contrário de sobrecarregar funções`

Reply:
maybe `estas são pensadas`

Reply:
not clear about `o a combinação de padrões`, maybe just `a combinação` ?

Reply:
`definiçoes` => `definições`

`com o mesmo` => `com a mesma`

Reply:
it was supposed to be `ou a combinação de padrões` but if it makes more sense to just say `a combinação` i can change that